documentation/source/users/rmg/liquids.rst

```diff
@@ -162,34 +162,25 @@ where `A, B, E, S` and `L` are the same solute descriptors used in the Abraham m
 :math:`\Delta G_{\rm solv}^{*}`. The lowercase coefficients `c', a', b', e', s'` and `l'` depend only on the solvent and were obtained
 by fitting to experimental data.
 
-The solvent descriptors (`c, e, s, a, b, l, c', a', b', e', s', l'`) are largely treated as regressed empirical coefficients.
-Parameters are provided in RMG's database for the following solvents:
+RMG's database has a solute library that contains the solute descriptors of around 300 compounds and it is located at
+``RMG-database/input/solvation/libraries/solute.py`` in RMG-database. For the compounds that are not found in the
+solute library, the group additivity method is used to estimate the solute descriptors as described below under
+`Group additivity method for solute descriptor estimation`.
 
-#. acetonitrile
-#. benzene
-#. butanol
-#. carbontet
-#. chloroform
-#. cyclohexane
-#. decane
-#. dibutylether
-#. dichloroethane
-#. dimethylformamide
-#. dimethylsulfoxide
-#. dodecane
-#. ethanol
-#. ethylacetate
-#. heptane
-#. hexadecane
-#. hexane
-#. isooctane
-#. nonane
-#. octane
-#. octanol
-#. pentane
-#. toluene
-#. undecane
-#. water
+The solvent descriptors (`c, e, s, a, b, l, c', a', b', e', s', l'`) are largely treated as regressed empirical coefficients.
+The solvent descriptors are fitted using in-house experimental solute parameter, solvation free energy, and solvation
+enthalpy data by Chung et al. ([Chung2021]_). The Abraham solvent parameters (`c, e, s, a, b, v, l`) are provided
+in RMG's database for 196 pure solvents and 3 binary mixture solvents for solvation free energy calculations. The Mintz
+solvent parameters (`c', a', b', e', s'`) are provided for 69 pure solvents in RMG's database for solvation enthalpy calculations.
+A full list of solvents and their solvent parameters can be found on (1) `the RMG-database git repository <https://github.com/ReactionMechanismGenerator/RMG-database/blob/master/input/solvation/libraries/solvent.py>`_
+or (2) `the RMG website <https://rmg.mit.edu/database/solvation/libraries/solvent/>`_. If RMG-database is installed,
+these solvent information can be found found in ``RMG-database/input/solvation/libraries/solvent.py``.
+
+You can also search for a solvent with a SMILES string by using a sample ipython notebook located at
```

Comment:
"with a" => "by",  "by using" => "using"

Reply:
it's fixed in this fixup commit 2d234db.